odoo/content.md

```diff
@@ -143,6 +143,30 @@ services:
       - POSTGRES_USER=odoo
 ```
 
+Use a custom database name instead of `postgres`
+
+```yml
+version: '3.1'
+services:
+  web:
+    image: %%IMAGE%%:17.0
+    depends_on:
+      - mydb
+    ports:
+      - "8069:8069"
+    environment:
+      - HOST=mydb
+      - USER=odoo
+      - PASSWORD=myodoo
+    command: odoo -d customdb
```

Comment:
The following command works

```
command: odoo -i base -d customdb
```

Reply:
POSTGRES_DB for the database must be set to `postgres` and then the custom database is created and populated.